app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  @@renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
+
+  def render_markdown_as_html markup
+    raw @@renderer.render markup
```

Comment:
@nicholasjhenry had mentionned houndci.
It might do the job... Let me create a ticket for this.